src/garage/torch/policies/base.py

```diff
@@ -0,0 +1,54 @@
+import abc
+import numpy as np
+
+
+class Policy(abc.ABC):
+    """
+    Policy base class without Parameterzied.
+
+    Args:
+        env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+
+    """
+    def __init__(self, env_spec):
+        self._env_spec = env_spec
+
+    @abc.abstractmethod
+    def get_action(self, observation):
+        """Get action given observation."""
+        pass
+
+    def get_actions(self, observations):
+        """Get actions given observations."""
+        return np.stack([self.get_action(obs) for obs in observations])
+
+    @property
+    def vectorized(self):
```

Comment:
i think we should remove this flag and assume all policies implement get_actions(). if they can't be vectorized naturally, they can implement it with a for loop.

Reply:
Yeah, I agree that's the right way to go.

Reply:
fixed